src/brev/produserbaredokumenter.js

```diff
@@ -1,108 +1,112 @@
-  },
-  {
-    kode: 'GENERELT_FRITEKSTBREV_BRUKER',
-    term: 'Fritekstbrev til bruker'
-  },
-  {
-    kode: 'GENERELT_FRITEKSTBREV_ARBEIDSGIVER',
-    term: 'Fritekstbrev til virksomhet'
-  },
-  {
-    kode: 'GENERELT_FRITEKSTBREV_VIRKSOMHET',
-    term: 'Fritekstbrev til virksomhet'
-  },
-  {
-    kode: 'GENERELT_FRITEKSTVEDLEGG',
-    term: 'Fritekstvedlegg'
-  },
+    {
+        kode: 'INNVILGELSE_YRKESAKTIV',
+        term: 'Innvilgelse yrkesaktiv'
+    },
+    {
+        kode: 'MELDING_FORVENTET_SAKSBEHANDLINGSTID',
+        term: 'Melding om forventet saksbehandlingstid'
+    },
+    {
+        kode: 'MELDING_FORVENTET_SAKSBEHANDLINGSTID_SOKNAD',
+        term: 'Melding om forventet saksbehandlingstid'
+    },
+    {
+        kode: 'MELDING_FORVENTET_SAKSBEHANDLINGSTID_KLAGE',
+        term: 'Melding om forventet saksbehandlingstid'
+    },
+    {
+        kode: 'MANGELBREV_BRUKER',
+        term: 'Melding om manglende opplysninger til bruker'
+    },
+    {
+        kode: 'MANGELBREV_ARBEIDSGIVER',
+        term: 'Melding om manglende opplysninger til virksomhet'
+    },
+    {
+        kode: 'INNVILGELSE_FOLKETRYGDLOVEN_2_8',
+        term: 'Vedtak om frivillig medlemskap'
+    },
+    {
+        kode: 'ORIENTERING_ANMODNING_UNNTAK',
+        term: 'Orienteringsbrev om anmodning unntak'
+    },
+    {
+        kode: 'MELDING_MANGLENDE_OPPLYSNINGER',
+        term: 'Melding om manglende opplysninger'
+    },
+    {
+        kode: 'MELDING_HENLAGT_SAK',
+        term: 'Melding om henlagt sak'
+    },
+    {
+        kode: 'ATTEST_A1',
+        term: 'Attest A1'
+    },
+    {
+        kode: 'AVSLAG_YRKESAKTIV',
+        term: 'Avslag eller orientering om anmodning om unntak'
+    },
+    {
+        kode: 'ANMODNING_UNNTAK',
+        term: 'Anmodning om unntak'
+    },
+    {
+        kode: 'INNVILGELSE_ARBEIDSGIVER',
+        term: 'Innvilgelse Arbeidsgiver'
+    },
+    {
+        kode: 'AVSLAG_ARBEIDSGIVER',
+        term: 'Avslag Arbeidsgiver'
+    },
+    {
+        kode: 'AVSLAG_MANGLENDE_OPPLYSNINGER',
+        term: 'Avslag manglende opplysninger'
+    },
+    {
+        kode: 'INNVILGELSE_YRKESAKTIV_FLERE_LAND',
+        term: 'Innvilgelse yrkesaktiv i flere land'
+    },
+    {
+        kode: 'ORIENTERING_VIDERESENDT_SOEKNAD',
+        term: 'Orienteringsbrev til bruker om videresendt søknad'
+    },
+    {
+        kode: 'VIDERESENDT_SOEKNAD_UTLAND',
+        term: 'Videresendt søknad til utenlandsk myndighet'
+    },
+    {
+        kode: 'ORIENTERING_UTPEKING_UTLAND',
+        term: 'Orienteringsbrev om utpeking til et annet land'
+    },
+    {
+        kode: 'TRYGDEAVTALE_GB',
+        term: 'Vedtaksbrev innvilgelse og Attest for NO/UK 1'
+    },
+    {
+        kode: 'TRYGDEAVTALE_US',
+        term: 'Vedtaksbrev innvilgelse og Attest for N/USA 2'
+    },
+    {
+        kode: 'TRYGDEAVTALE_CAN',
+        term: 'Vedtaksbrev innvilgelse og Attest for NO/CAN 2'
+    },
```

Comment:
Hva er N/USA 2? Burde det stå NO/USA som de andre to?

Reply:
Vi justerer evt. term for TRYGDEAVTALE_US i egen PR